project/users/management/commands/create_user.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.contrib.auth.models import Group
+
+class Command(BaseCommand):
+    help = 'Create user.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('name', type=str)
+        parser.add_argument('email', type=str)
+        parser.add_argument('password', type=str)
+        parser.add_argument('team', type=str)
+
+    def handle(self, *args, **kwargs):
+        name = kwargs['name']
+        email = kwargs['email']
+        password = kwargs['password']
```

Comment:
Passwords should better be received from stdin, otherwise they'll be stored in the command history. See [`createsuperuser` source code](https://github.com/django/django/blob/master/django/contrib/auth/management/commands/createsuperuser.py#L120) for implementation.

Reply:
Thanks for spotting this, great! 
I think that only reading from input would be enough, because actually that translates to a createuser call, which does all the validation and staff (except that the give password twice, which is not quite useful for us, because IT members create the accounts. If I am missing something, feel free to reopen this!